src/platform/i386/kernel.c

```diff
@@ -163,6 +163,7 @@ kmain(struct multiboot *mboot, u32_t mboot_magic, u32_t esp)
 	kern_boot_comp(INIT_CORE);
 	lapic_init();
 	hpet_init();
+	chal_irq_enable(HW_SERIAL, 0);
```

Comment:
This should really be a function in serial.c. Like "serial_late_init()" or "serial_irq_init()" something like that.